### PR TITLE
ozone adapter  2.1 - bug fix for multi bids + GDPR parameter handling

### PR DESCRIPTION
**Pull request by afsheenb**
## Type of change
- [X] Bugfix
- [X] Feature

## Description of change

*bug fix for single request with multiple bids.
*parameters for handling GDPR requests inside regs.ext object

- engineering@ozoneproject.com

### CODE REVIEW

File: modules/ozoneBidAdapter.md [hunk 1]
```diff
@@ -36,8 +36,7 @@ adUnits = [{
                             publisherId: 'OZONENUK0001', /* an ID to identify the publisher account  - required */
                             siteId: '4204204201', /* An ID used to identify a site within a publisher account - required */
                             placementId: '0420420421', /* an ID used to identify the piece of inventory - required - for appnexus test use 13144370. */
-							customData": [{"settings": {}, "targeting": {"key": "value", "key2": ["value1", "value2"],}}] /* optional array with 'targeting' placeholder for passing publisher specific key-values for targeting. */                            
-							ozoneData: {"key1": "value1", "key2": "value2"}, /* optional JSON placeholder for for passing ozone project key-values for targeting. */
```
Comment (jsnellbaker) [inline modules/ozoneBidAdapter.md]: Can you clarify on the change to remove this property `ozoneData` from the md file?  Was it intentional? 

It's still present in the adapter code from what I can tell.  It's also present on the prebid.org site (under http://prebid.org/dev-docs/bidders.html#ozone).

Comment (afsheenb) [reply modules/ozoneBidAdapter.md]: Yes, this was intentional. We'll be submitting a pull request to remove it from the prebid.org documentation shortly.

**Comment by afsheenb** [comment]
Answers:

• references to ozoneData have now been removed from our adapter code, md file and spec tests.

• the quotes around the customData array in the md file have been removed - thanks for flagging. The quotes in the array for 'key2' in the example are intentional as they are meant to be a array of strings.

• we will submit updates for the documentation on prebid.org to remove references to ozoneData.

• we've also added some more stringent checks for the GDPR bits we are passing up.

File: modules/ozoneBidAdapter.md [hunk 2]
```diff
@@ -36,8 +36,7 @@ adUnits = [{
...
+							customData: "[{"settings": {}, "targeting": {"key": "value", "key2": ["value1", "value2"]}}]",/* optional array with 'targeting' placeholder for passing publisher specific key-values for targeting. */
```
[the remark this answers is not in the document]
Comment (afsheenb) [reply modules/ozoneBidAdapter.md]: • the quotes around the customData array in the md file have been removed - thanks for flagging. The quotes in the array for 'key2' in the example are intentional as they are meant to be a array of strings.

**Comment by AskRupert-DM** [comment]
let us know if you have any further questions @jsnellbaker following on from @afsheenb's feedback - thanks in advance!

**Comment by AskRupert-DM** [comment]
Hey @jsnellbaker - can you confirm what documentation are you are waiting on ? 

The .md file was updated at the same time the last set of code changes were submitted & the reference to ozone data was removed from this page - http://prebid.org/dev-docs/bidders.html#ozone

**Comment by jsnellbaker** [comment]
@AskRupert-DM Sorry about that - I didn't notice the docs PR was already merged.  We should be good at this point.

**Comment by AskRupert-DM** [comment]
No worries... thanks @jsnellbaker !

**Pull request merged**